package.json

```diff
@@ -48,7 +48,7 @@
         "nonce": "^1.0.4",
         "redis": "^2.8.0",
         "request": "^2.88.0",
-        "stratum-pool": "git+https://github.com/s-nomp/node-stratum-pool.git"
+        "stratum-pool": "git+https://github.com/globaltoken/snomp-stratum-pool.git"
```

Comment:
This isn't an option. You'll need to create a PR to https://github.com/s-nomp/node-stratum-pool.git to support this coin and restore the original value.